core/templates/dev/head/pages/learner_dashboard/LearnerDashboard.js

```diff
@@ -293,9 +457,19 @@ oppia.controller('LearnerDashboard', [
         $scope.completedToIncompleteCollections = (
           responseData.completed_to_incomplete_collections
         );
+        var threadSummaryDicts = responseData.thread_summaries;
+        $scope.threadSummaries = [];
+        for (var index = 0; index < threadSummaryDicts.length; index++) {
+          $scope.threadSummaries.push(
+            FeedbackThreadSummaryObjectFactory.createFromBackendDict(
+              threadSummaryDicts[index]));
+        }
+        $scope.numberOfUnreadThreads = responseData.number_of_unread_threads;
 
-        $scope.activeSection = LEARNER_DASHBOARD_SECTIONS.INCOMPLETE;
-        $scope.activeSubSection = LEARNER_DASHBOARD_SUBSECTIONS.EXPLORATIONS;
+        $scope.activeSection = LEARNER_DASHBOARD_SECTION_I18N_IDS.INCOMPLETE;
+        $scope.activeSubSection = (
```

Comment:
Remember, here and elsewhere: "subsection" is one word, so this should be activeSubsection. There are other instances of this.